### PR TITLE
Change to 'check if someone can rent your residential property' smart answer

### PR DESCRIPTION
**Pull request by callumknights**
Added new question and outcomes for the 'check if someone can rent your
residential property' smart answer.

Trello: https://trello.com/c/GBxr4ew1/2347-3-content-updates-to-check-if-someone-can-rent-your-residential-property

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: lib/smart_answer_flows/landlord-immigration-check.rb [hunk 1]
```diff
@@ -189,16 +189,32 @@ def define
...
 
+      # new Q15
+      radio :from_aus_canada? do
+        option "yes"
+        option "no"
+
+        next_node do |response|
+          case response
+          when "yes"
+            outcome :outcome_can_rent_check_arrived_lasy_six_months
```
Comment (thomasleese) [inline lib/smart_answer_flows/landlord-immigration-check.rb:211]:
```suggestion
            outcome :outcome_can_rent_check_arrived_last_six_months
```

Comment (callumknights) [reply lib/smart_answer_flows/landlord-immigration-check.rb:211]: Good spot 👍

**Review by thomasleese — APPROVED** [review]

**Comment by theseanything** [comment]
@callumknights is this work still progressing?

**Comment by callumknights** [comment]
> @callumknights is this work still progressing?

This was held up in fact check for a while, a very long while really. I'll drop content a message and see if they have heard anything.

**Comment by kevindew** [comment]
This seems to have gone stale, can always re-open if work is still needed.

**Pull request closed without merging**